### PR TITLE
Display queries in app mobile

### PR DESCRIPTION
**Pull request by ger-benjamin**
Ref to https://github.com/camptocamp/c2cgeoportal/issues/1800

Example: https://ger-benjamin.github.io/ngeo/master/examples/contribs/gmf/apps/mobile/index.html

I don't currently use ng-swipe (for "gesture" actions) (further PR)

### CODE REVIEW

File: contribs/gmf/apps/appmodule.js [hunk 1]
```diff
@@ -11,4 +11,4 @@ goog.require('gmf');
 /**
  * @type {!angular.Module}
  */
-app.module = angular.module('app', [gmf.module.name]);
+app.module = angular.module('app', [gmf.module.name, 'ngAnimate']);
```
Comment (ger-benjamin) [inline contribs/gmf/apps/appmodule.js]: I'm not sur that that's the good place to do this...

File: contribs/gmf/src/controllers/abstract.js [hunk 2]
```diff
@@ -81,6 +84,8 @@ gmf.AbstractController = function(config, $scope, $injector) {
    */
   this.queryActive = true;
 
+  $injector.get('gmfQueryManager');
```
Comment (ger-benjamin) [inline contribs/gmf/src/controllers/abstract.js]: Not used but it must be injected.

Comment (tsauerwein) [inline contribs/gmf/src/controllers/abstract.js]: Maybe you want to add this as a comment in the code.

File: contribs/gmf/apps/appmodule.js (hunk 1 above)
Comment (sbrunner) [inline contribs/gmf/apps/appmodule.js]: For me it should be in the gmf Module...

File: contribs/gmf/less/displayqueries.less [hunk 3]
```diff
@@ -0,0 +1,151 @@
+.mobiledisplayqueries {
```
Comment (sbrunner) [inline contribs/gmf/less/displayqueries.less]: shouldn't the file be named `mobiledisplayqueries.less` ?

File: contribs/gmf/src/controllers/abstract.js [hunk 4]
```diff
@@ -81,6 +84,9 @@ gmf.AbstractController = function(config, $scope, $injector) {
    */
   this.queryActive = true;
 
+  // Not used in this file but the QueryManager must be injected to be watched.
+  $injector.get('gmfQueryManager');
```
Comment (sbrunner) [inline contribs/gmf/src/controllers/abstract.js]: The goal is a king of assert?

Comment (ger-benjamin) [inline contribs/gmf/src/controllers/abstract.js]: Not really. If it's not injected, there is nothing... literally nothing. I you inject it, the value can be watched and something can happens. but it not like a test...

**Comment by ger-benjamin** [comment]
Validated

**Pull request merged**